Representable/representable
# shorten slug to <35 char

**changes**
- shortens organization and drive slugs to be less than 35 characters, split only at spaces (should be more readable!)

**to test**
- create a new org with a long name
- check slug makes sense and is short enough
- create a drive within said org
- check slug looks good!

can also shorten the slugs even more if we'd like to make them more readable. i chose 35 char as an arbitrary limit that seemed to give enough length while not being too verbose
